lib/realtime/alerts_fetcher.ex

```diff
@@ -0,0 +1,133 @@
+defmodule Realtime.AlertsFetcher do
+  use GenServer
+  require Logger
+  alias Schedule.Route
+
+  @default_poll_interval_ms 3 * 60 * 1_000
```

Comment:
Yes, that's a good point. Of course, if streaming had been a more reasonable approach for implementing this feature it would have provided better latency. In any event it should be a fairly inexpensive API query so I'm okay with bumping it to once / minute.